proto/desmos/profiles/v1beta1/query.proto

```diff
@@ -46,16 +46,10 @@ service Query {
         "/desmos/relationships/v1beta1/blocks/{user}";
   }
 
-  // ChainLinks queries all the links for all users
-  rpc ChainLinks(QueryChainLinksRequest) returns (QueryChainLinksResponse) {
+  // ProfileByChainLink the profile for the user having the given address and chain name
+  rpc ProfileByChainLink(QueryProfileByChainLinkRequest) returns (QueryProfileByChainLinkResponse) {
     option (google.api.http).get =
-        "/desmos/relationships/v1beta1/links";
-  }
-
-  // UserChainLinks queries all the links for the user having the given address
-  rpc UserChainLinks(QueryUserChainLinksRequest) returns (QueryUserChainLinksResponse) {
-    option (google.api.http).get =
-        "/desmos/relationships/v1beta1/links/{user}";
+      "/desmos/profiles/v1beta1/profiles/{chain_name}/{target}";
```

Comment:
I think it would be better to rename this `chain-links` instead. Also using `target_address` might be better: 

```suggestion
      "/desmos/profiles/v1beta1/chain-links/{chain_name}/{target_address}";
```